src/internal/module/update.go

```diff
@@ -53,11 +57,15 @@ func (p Module) getSemVerTagsFromRSS() ([]string, error) {
 func (p Module) getLastSemVerTag() (string, error) {
 	semverTags, err := p.getSemVerTagsFromRSS()
 	if err != nil {
-		return "", err
+		p.Logger.Error("Unable to fetch tags, skipping", slog.Any("err", err))
+		return "", nil
+		//return "", err
```

Comment:
leftover comment

Reply:
I did this intentionally as I see it as a stopgap

Reply:
Could you add a TODO comment then?